man/kernel-install.xml

```diff
@@ -111,7 +111,7 @@
 
             <listitem><para><filename>90-uki-copy.install</filename> copies a file
             <filename>uki.efi</filename> from <varname>$KERNEL_INSTALL_STAGING_AREA</varname> or if it does
-            not exist the <replaceable>KERNEL-IMAGE</replaceable> argument, iff it has a
+            not exist the <replaceable>KERNEL-IMAGE</replaceable> argument, if it has a
```

Comment:
This is not a typo. But this misunderstanding happens a lot in the docs. iff is mathematician slang for "if and only if". Since we kept this around, this should stay, too.

Reply:
Using math slang in the docs doesn't look like a good idea to me.

Care to explain what exactly the semantic difference is?  I read both versions as only kernel images named *.efi being accepted.

Reply:
Just saw that this actually is the last userfacing "iff", the rest is only in comments, so this should go as well in the interest of being jargon free. A valid subsitution could be 
```suggestion
            not exist the <replaceable>KERNEL-IMAGE</replaceable> argument, only if it has a
```

Logically this means if is a one way implication and iff is bidrectional. In both cases "The file has a .efi suffix, so it will be copied to the directory" is logically valid, but "The file has been copied to the directory, so it must have a .efi suffix" is only true in the latter not the former.